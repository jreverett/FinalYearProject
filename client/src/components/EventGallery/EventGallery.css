.event-gallery-masonry {
  display: flex;
  flex-flow: row wrap;
  margin: 20px;
  width: 100%;
}
