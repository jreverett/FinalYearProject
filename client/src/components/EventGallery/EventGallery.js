import React, { Component, Fragment } from 'react';
import { Button, CardDeck, CardColumns } from 'react-bootstrap';

import { EventTile } from '../index';
import './EventGallery.css';

class EventGallery extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    var events = this.props.events.data;
    var eventTiles = [];

    if (events) {
      eventTiles = events.map(function(event, index) {
        return <EventTile key={index} eventDetails={event} />;
      });
    }

    return <CardColumns className="card-column">{eventTiles}</CardColumns>;
  }
}

export default EventGallery;
