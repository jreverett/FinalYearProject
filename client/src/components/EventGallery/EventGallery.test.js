import React from 'react';
import renderer from 'react-test-renderer';
import EventGallery from './EventGallery';

describe('<EventGallery />', () => {
  it('matches the snapshot', () => {
    const testEvents = [];
    const tree = renderer.create(<EventGallery events={testEvents} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
