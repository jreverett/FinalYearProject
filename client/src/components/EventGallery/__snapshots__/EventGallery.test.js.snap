// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventGallery /> matches the snapshot 1`] = `
<div
  className="event-gallery-masonry"
/>
`;
